src/inbox/mod_inbox.erl

```diff
@@ -78,6 +78,33 @@
 
 -export_type([get_inbox_params/0]).
 
+%%--------------------------------------------------------------------
+%% gdpr callbacks
+%%--------------------------------------------------------------------
+
+-behaviour(gdpr).
```

Comment:
It breaks the convention where we have behaviour and exports at the top of the file.

Reply:
I'll move to the top of the file.